packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.html

```diff
@@ -0,0 +1,105 @@
+            data-translate="telephony_line_softphone_add_device_description_bold"
+        ></b>
+        <span
+            data-translate="telephony_line_softphone_add_device_description"
+        ></span>
+    </p>
+    <div
+        class="d-flex flex-wrap w-100 minw-0 align-items-center justify-content-between"
+    >
+        <form class="form-inline" name="$ctrl.softphoneAddDeviceForm">
+            <div class="row">
+                <oui-field
+                    class="col-md-12"
+                    data-help-text="{{:: 'telephony_line_softphone_generate_warning' | translate }}"
+                    label="{{ ::'telephony_line_softphone_add_device_label' | translate }}"
+                    size="sm"
+                >
+                    <div class="input-group">
+                        <input
+                            type="text"
+                            id="input_name"
+                            name="input_name"
+                            class="oui-input"
+                            data-ng-model="$ctrl.input.name"
+                            data-ng-pattern="/^[a-zA-Z0-9\-\_]+$/"
+                            data-ng-class="{'oui-field_error': $ctrl.softphoneAddDeviceForm.input_name.$invalid}"
+                            data-translate-attr="{ placeholder: 'telephony_line_softphone_device_name_placeholder' }"
+                            required
+                        />
+                        <div class="input-group-btn">
+                            <oui-button
+                                on-click=":: $ctrl.checkInputAndGenerateLink()"
+                                data-variant="primary"
+                            >
+                                <span
+                                    data-translate="telephony_line_softphone_generate_link"
+                                >
+                                </span>
+                            </oui-button>
+                        </div>
+                    </div>
+                </oui-field>
+            </div>
+            <div class="row">
+                <oui-field
+                    class="col-md-12"
+                    label="{{ ::'telephony_line_softphone_record_link' | translate }}"
+                    size="sm"
+                >
+                    <div class="input-icons-softphone-record-link">
+                        <input
+                            type="text"
+                            id="input_record_link"
+                            name="input_record_link"
+                            data-ng-readonly="true"
+                            class="oui-input"
+                            required
+                            data-ng-model="$ctrl.input.recordLink"
+                        />
+                        <i
+                            class="oui-icon oui-icon-copy-normal"
+                            aria-hidden="true"
+                        >
+                        </i>
+                    </div>
```

Comment:
You should use the ui-kit clipboard component.
https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-clipboard-webcomponent--default